scrap-test.txt

```diff
@@ -34,20 +34,8 @@ You are now connected to database "nc_games_test" as user "patrickdunn".
- philippaclaire9 | Jenga                                                |         2 | dexterity        | https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png                                       | 2021-01-18 10:01:41.251 |     5 | Leslie Scott           |             3
- bainesface      | Ultimate Werewolf                                    |         3 | social deduction | https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png                                       | 2021-01-18 10:01:41.251 |     5 | Akihisa Okui           |             3
- mallionaire     | That's just what an evil person would say!           |        11 | social deduction | https://images.pexels.com/photos/163064/play-stone-network-networked-interactive-163064.jpeg                    | 2021-01-18 10:01:41.251 |     8 | Fiona Lohoar           |             0
- mallionaire     | One Night Ultimate Werewolf                          |         8 | social deduction | https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260 | 2021-01-18 10:01:41.251 |     5 | Akihisa Okui           |             0
- mallionaire     | Build you own tour de Yorkshire                      |        10 | social deduction | https://images.pexels.com/photos/258045/pexels-photo-258045.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260   | 2021-01-18 10:01:41.251 |    10 | Asger Harding Granerud |             0
- mallionaire     | Agricola                                             |         1 | euro game        | https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png                                       | 2021-01-18 10:00:20.514 |     1 | Uwe Rosenberg          |             0
- mallionaire     | Proident tempor et.                                  |         5 | social deduction | https://images.pexels.com/photos/163064/play-stone-network-networked-interactive-163064.jpeg                    | 2021-01-07 09:06:08.077 |     5 | Seymour Buttz          |             0
- mallionaire     | Occaecat consequat officia in quis commodo.          |         6 | social deduction | https://images.pexels.com/photos/278918/pexels-photo-278918.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260   | 2020-09-13 15:19:28.077 |     8 | Ollie Tabooger         |             0
- mallionaire     | Settlers of Catan: Don't Settle For Less             |        13 | social deduction | https://images.pexels.com/photos/163064/play-stone-network-networked-interactive-163064.jpeg                    | 1970-01-10 03:08:38.4   |    16 | Klaus Teuber           |             0
-(13 rows)
+ review_id |  title   | category  |   designer    |    owner    |  review_body  |                              review_img_url                               |       created_at        | votes 
```

Comment:
I wouldn't commit your scrap file, it's better practice not to git add -a and just the files you've changed